<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="Cont_Titulo" native="true">
      <property name="maximumSize">
       <size>
        <width>10000</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 85, 127);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="Layout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="Imagen1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../Archivos/Recursos.qrc">:/Logos/UAT.jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="Ltitulo">
           <property name="minimumSize">
            <size>
             <width>800</width>
             <height>0</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>2</width>
             <height>2</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>36</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Proyecto: Granja Inteligente &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="Imagen2">
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../Archivos/Recursos.qrc">:/Logos/Facultad.jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="Cont_Conectar" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="Bconectar">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string extracomment="Presionar despues de haber seleccionado el puerto"/>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;
border-radius: 5px;
padding: 5px;
</string>
         </property>
         <property name="text">
          <string>CONECTAR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="CBpuertos">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;
border-radius: 5px;
padding: 5px;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="Cont_Libre" native="true">
      <widget class="QStackedWidget" name="Stacked_paginas">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>991</width>
         <height>531</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(206, 255, 224);</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>561</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Servicio de riego (Medici√≥n de temperatura en grados celcius )</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Bmostrar1">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>420</y>
           <width>100</width>
           <height>28</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string extracomment="Presionar despues de haber seleccionado el puerto"/>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;
border-radius: 5px;
padding: 5px;
</string>
         </property>
         <property name="text">
          <string>PRENDER</string>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>641</width>
           <height>331</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="QVcanvas1"/>
        </widget>
        <widget class="QPushButton" name="Bguardar1">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>420</y>
           <width>100</width>
           <height>28</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string extracomment="Presionar despues de haber seleccionado el puerto"/>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;
border-radius: 5px;
padding: 5px;
</string>
         </property>
         <property name="text">
          <string>Guardar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Bactuador1">
         <property name="geometry">
          <rect>
           <x>780</x>
           <y>450</y>
           <width>200</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string extracomment="Presionar despues de haber seleccionado el puerto"/>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;
border-radius: 5px;
padding: 5px;
</string>
         </property>
         <property name="text">
          <string>Activar actuador</string>
         </property>
        </widget>
        <widget class="QListWidget" name="lista_datos1">
         <property name="geometry">
          <rect>
           <x>660</x>
           <y>50</y>
           <width>181</width>
           <height>331</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>840</x>
           <y>410</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Rociador</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 224, 255);</string>
        </property>
        <widget class="QPushButton" name="Bmostrar2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>420</y>
           <width>100</width>
           <height>28</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string extracomment="Presionar despues de haber seleccionado el puerto"/>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;
border-radius: 5px;
padding: 5px;
</string>
         </property>
         <property name="text">
          <string>PRENDER</string>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>641</width>
           <height>331</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="QVcanvas2"/>
        </widget>
        <widget class="QListWidget" name="lista_datos2">
         <property name="geometry">
          <rect>
           <x>660</x>
           <y>50</y>
           <width>181</width>
           <height>331</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="Bguardar2">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>420</y>
           <width>100</width>
           <height>28</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string extracomment="Presionar despues de haber seleccionado el puerto"/>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;
border-radius: 5px;
padding: 5px;
</string>
         </property>
         <property name="text">
          <string>Guardar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Bactuador2">
         <property name="geometry">
          <rect>
           <x>780</x>
           <y>450</y>
           <width>200</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string extracomment="Presionar despues de haber seleccionado el puerto"/>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;
border-radius: 5px;
padding: 5px;
</string>
         </property>
         <property name="text">
          <string>Activar actuador</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>461</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Servicio de luz (Medici√≥n de nivel de iluminaci√≥n)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>840</x>
           <y>410</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>LED</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 215, 215);</string>
        </property>
        <widget class="QListWidget" name="lista_datos3">
         <property name="geometry">
          <rect>
           <x>660</x>
           <y>50</y>
           <width>181</width>
           <height>331</height>
          </rect>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>641</width>
           <height>331</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="QVcanvas3"/>
        </widget>
        <widget class="QPushButton" name="Bmostrar3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>420</y>
           <width>100</width>
           <height>28</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string extracomment="Presionar despues de haber seleccionado el puerto"/>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;
border-radius: 5px;
padding: 5px;
</string>
         </property>
         <property name="text">
          <string>PRENDER</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Bguardar3">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>420</y>
           <width>100</width>
           <height>28</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string extracomment="Presionar despues de haber seleccionado el puerto"/>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;
border-radius: 5px;
padding: 5px;
</string>
         </property>
         <property name="text">
          <string>Guardar</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Bactuador3">
         <property name="geometry">
          <rect>
           <x>780</x>
           <y>450</y>
           <width>200</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>35</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string extracomment="Presionar despues de haber seleccionado el puerto"/>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;
border-radius: 5px;
padding: 5px;
</string>
         </property>
         <property name="text">
          <string>Activar actuador</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>561</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Servicio de seguridad (Sensor de proximidad medido en cm)</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>840</x>
           <y>410</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Buzzer</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>1020</x>
         <y>0</y>
         <width>119</width>
         <height>181</height>
        </rect>
       </property>
       <property name="title">
        <string>Servicios</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="RBpagina1">
          <property name="styleSheet">
           <string notr="true">border: 2px solid black;
border-radius: 5px;
padding: 5px;
background-color: rgb(206, 255, 224);</string>
          </property>
          <property name="text">
           <string>Riego</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RBpagina2">
          <property name="styleSheet">
           <string notr="true">border: 2px solid black;
border-radius: 5px;
padding: 5px;
background-color: rgb(203, 224, 255);</string>
          </property>
          <property name="text">
           <string>Luz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RBpagina3">
          <property name="styleSheet">
           <string notr="true">border: 2px solid black;
border-radius: 5px;
padding: 5px;
background-color: rgb(255, 215, 215);</string>
          </property>
          <property name="text">
           <string>Alarma</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="Bconfiguracion">
       <property name="geometry">
        <rect>
         <x>1020</x>
         <y>190</y>
         <width>120</width>
         <height>35</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string extracomment="Presionar despues de haber seleccionado el puerto"/>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid black;
background-color: rgb(204, 201, 223);
border-radius: 5px;
padding: 5px;
</string>
       </property>
       <property name="text">
        <string>Configuraci√≥n</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="Cont_Inferior" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <layout class="QHBoxLayout" name="_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="Imagen3">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../Archivos/Recursos.qrc">:/Logos/Castor Ingeniero.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../Archivos/Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
